packages/admin-panel/src/pages/resources/CustomLandingPagesPage.js

```diff
@@ -8,9 +9,9 @@ import { prettyArray } from '../../utilities';
 const LANDING_PAGES_ENDPOINT = 'landingPages';
 
 // the URL prefix to display in the url_segment field
-const URL_PREFIX = `${window.location.origin
-  .replace('admin', 'www')
-  .replace(new RegExp('(https://)|(http://)'), '')}/`;
+const URL_PREFIX = window.location.origin.replace('admin', 'www');
+
+const DISPLAY_URL_PREFIX = `${URL_PREFIX.replace(new RegExp('(https://)|(http://)'), '')}/`;
 
 // All the fields for create/edit of a custom landing page
 const FIELDS = [
```

Comment:
Suggestion:
FIELDS lists fields flat { name: ..., code: ... }

EDIT_FIELDS has nesting and plucks from FIELDS e.g. [type: section, fields: [FIELDS.name, FIELDS.code]] or default [...Object.values(FIELDS)]

COLUMNS plucks from FIELDS e.g. [FIELDS.name, FIELDS.code] or default [...FIELDS]

Decorating FIELDS inline to reorder columns etc. is a great idea, my suggestion would be to keep it outside because in the future we will get FIELDS straight from the db and largely autogenerate the admin panel.